template/java12/bin/template.yml

```diff
@@ -0,0 +1 @@
+language: java8
```

Comment:
Shouldn't it be changed to java12?

Reply:
The bin folder should be removed. This file isnt used